packages/editor/src/components/projectEditor/ProjectEditor.tsx

```diff
@@ -23,7 +23,7 @@ import TopBar from '../topbar';
 import BottomBar from './bottomBar';
 import ContactContainer from '../contactContainer';
 import { PreviewPanel, TransactionLogPanel, OutputPanel, Explorer, InteractPanel, ConfigurePanel } from './panels';
-import { IconTransactions, IconFolderOpen, IconInteract, IconConfigure, IconDownloadDApp, IconTogglePreview } from '../icons';
```

Comment:
Please revert this as I left it like that on purpose (for the user, the preview panel is the combination of both the browser and the transactions).